code/datums/brain_damage/severe.dm

```diff
@@ -0,0 +1,193 @@
+//Severe traumas, when your brain gets abused way too much.
+//These range from very annoying to completely debilitating.
+//They cannot be cured with chemicals, and require brain surgery to solve.
+
+/datum/brain_trauma/severe
+
+/datum/brain_trauma/severe/mute
+	name = "Mutism"
+	desc = "Patient is completely unable to speak."
+	scan_desc = "extensive damage to the brain's language center"
+	gain_text = "<span class='warning'>You forget how to speak!</span>"
+	lose_text = "<span class='notice'>You suddenly remember how to speak.</span>"
+
+/datum/brain_trauma/severe/mute/on_gain()
+	owner.disabilities |= MUTE
+	..()
+
+//no fiddling with genetics to get out of this one
+/datum/brain_trauma/severe/mute/on_life()
+	if(!(owner.disabilities & MUTE))
+		on_gain()
+	..()
+
+/datum/brain_trauma/severe/mute/on_lose()
+	owner.disabilities &= ~MUTE
```

Comment:
What if you gain mute from another source like genetics then this happens

Reply:
How about you route the ability to hear messages through the brain?  This would also allow for the ability for a brain trauma that sutbtly changes messages.  Such as one that replaces locations with other locations.

Reply:
Would require a bit of refactoring, but you could do disabilities like martial arts where there are regular ones and temporary ones that take priority, allowing you to add and remove temporary ones without affecting normal ones

Reply:
now if only someone would merge that